resources/web/style/admonishment.pcss

```diff
@@ -0,0 +1,38 @@
+#guide .Admonishment {
+  position: relative;
+  font-size: 16px;
+
+  &:hover {
+    color: black;
+    transition: all 0.3s ease;
+    .Admonishment-detail {
+      z-index: 10000;
+      left: -5em;
+      opacity: 1;
+      color: #555;
+      transition: color 0.5s ease;
+    }
+  }
+
+  &-title, &-version {
```

Comment:
Oh, that's cool!

Reply:
I like it! Nice and simple way to contain the component. It doesn't *always* work as you can see inside the style that is inside the hover.